schema/switch.yml

```diff
@@ -242,3 +242,294 @@ properties:
+          type: boolean
+          default: false
+          enum: 
+            - true
+            - false
+        dhcp-snoop-inf-opt-remoteid:
+          description: "It specifies the remote ID sub-option in option 82, which typically includes information like the circuit ID or remote host identifier"
+          type: string
+          maxLength: 32
+          minLength: 1
+        dhcp-snoop-inf-opt-policy:
+          description: "This defines the policy for handling packets with option 82, determining whether they should be forwarded or dropped based on the configuration"
+          type: string
+          enum: 
+            - drop
+            - keep
+            - replace
+  acl:
+    description: Contains all the access control rule definitions
+    type: array
+    items:
+      type: object
+      properties:
+        acl-type:
+          description: Type of the access control list
+          type: string
+          enum:
+            - ipv4
+            - ipv6
+            - ipv4Ext
+            - ipv6Ext
+            - mac
+            - arp
+        acl-name:
+          description: The identifier or name for the Access Control List
+          type: string
+          maxLength: 32
+          minLength: 1
+        acl-rule-action:
+          description: Defines whether to permit or deny traffic matching the rule
+          type: string
+          enum:
+            - permit
+            - deny
+        acl-source-macaddress:
+          description: Specifies the source MAC address to filter on
+          type: string
+          format: uc-mac
+        acl-source-macbitmask:
+          description: The mask applied to the source MAC address
+          type: string
+          format: uc-mac
+        acl-dest-macaddress:
+          description: Specifies the destination MAC address for the filter
+          type: string
+          format: uc-mac
+        acl-dest-macbitmask:
+          description: The mask applied to the destination MAC address
+          type: string
+          format: uc-mac
+        acl-packet-format:
+          description: Identifies the protocol encapsulated in the Ethernet frame by its EtherType
+          type: string
+          enum:
+            - any
+            - untagged-eth2
+            - untagged-802.3
+            - tagged-eth2
+            - tagged-802.3
+        acl-vlanid:
+          description: Specifies a VLAN ID to filter traffic from a specific VLAN
+          type: integer
+          minimum: 1
+          maximum: 4094
+        acl-vid-bitmask:
+          description: The mask applied to the VLAN ID
+          type: integer
+          minimum: 1
+          maximum: 4095
+        acl-ethertype:
+          description: Filters packets based on the custom EtherType field in the Ethernet frame
+          type: string
+          default: "800"
+        acl-ethertype-bitmask:
+          description: The mask applied to the EtherType field
+          type: string
+          default: "FFFF"
```

Comment:
Do we really want to enforce underlying device / OLS implementation to do a hex-string parsing?
For VID we use integer masks

Reply:
Since the ethertype is defined in HEX during config, the mask as well came in that way. Did not think of the integer masks to be put here since the primary parameter that is masked is defined to be HEX

Reply:
@binnyjeshan i do understand that HEX representation is a bit more common and more easily readable;
However, we're in process of defining M2M interface on top of uCentral transport protocol;
And i don't really think the data _presentation_ should be part of schema definition;
After all, the config is generated by SW and is being received / parsed by SW as well, hence integer repr is much more convinient to use here;

However, it's not a dealbreaker, so we can stick with HEX if that's easier / maybe there's already SW that uses hex-representation?

Reply:
OK @Cahb
I understand your view. Let's go with this for now, so that existing Vendors can interpret the incoming config as a HEX value represented in a string format just as how their current CLI/UI lets them do. 
I'll open this up in for our future dev calls, to discuss and think about how we can make it better.